cmd/utils/fdlimit_windows.go

```diff
@@ -0,0 +1,36 @@
+// Copyright 2016 The go-ethereum Authors
+// This file is part of go-ethereum.
+//
+// go-ethereum is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// go-ethereum is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with go-ethereum. If not, see <http://www.gnu.org/licenses/>.
+
+package utils
+
+// #include <stdio.h>
+import "C"
+
+// raiseFdLimit tries to maximize the file descriptor allowance of this process
+// to the maximum hard-limit allowed by the OS.
+func raiseFdLimit(max uint64) error {
+	if max > 2048 {
+		max = 2048 // windows limits
+	}
+	C._setmaxstdio(C.int(max))
```

Comment:
Further reading suggests that there is no limit when using win32 handles (see [this SO answer](http://stackoverflow.com/questions/870173/is-there-a-limit-on-number-of-open-files-in-windows/4276338#4276338) and the [linked mysql ticket](http://bugs.mysql.com/bug.php?id=24509)).

I would like to suggest that we use noop `raiseFdLimit` and return some constant from `getFdLimit`.